src/Network/NWBrowseResult.cs

```diff
@@ -0,0 +1,93 @@
+//
+// NWBrowseResult.cs: Bindings the Network nw_browse_result_t API.
+//
+// Authors:
+//   Manuel de la Pena (mandel@microsoft.com)
+//
+// Copyrigh 2019 Microsoft Inc
+//
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using ObjCRuntime;
+using Foundation;
+using CoreFoundation;
+
+using OS_nw_browse_result=System.IntPtr;
+using OS_nw_endpoint=System.IntPtr;
+using OS_nw_txt_record=System.IntPtr;
+
+namespace Network {
+
+	[Flags, TV (13,0), Mac (10,15), iOS (13,0), Watch (6,0)]
+	public enum NWBrowseResultChange : ulong {
+		Invalid = 0x00,
+		Identical = 0x01,
+		ResultAdded = 0x02,
+		ResultRemoved = 0x04,
+		TxtRecordChanged = 0x20,
+		InterfaceAdded = 0x08, 
```

Comment:
For maintenance, it's probably best to put these into numerical order rather than how they are documented.
For flags like this, I prefer to use shift constants like `Identical = 1 << 0, ResultAdded = 1 << 1, ResultRemoved = 1 << 2` etc.